README.md

```diff
@@ -123,22 +123,34 @@ All Wire translations are crowdsourced via [Crowdin](https://crowdin.com/project
 
 #### Add new strings
 
+**Info:**
+
+- To download translations we use Crowdin API v1, and you need to setup your [username](https://crowdin.com/settings#account) and [api_key](https://crowdin.com/settings#api-key) (Account API key).
+- To upload translations we use Crowdin CLI v3, and you will need to setup [project_identifier](https://crowdin.com/project/wire-webapp/settings#api) and [api_token](https://crowdin.com/settings#api-key) (Personal Access Token).
+
+**Setup:**
+
+Create a `keys/crowdin.yaml` in this repository and add the following entries:
+
+```yaml
+api_key: your-account-api-key
+api_token: your-personal-access-token
+project_identifier: wire-webapp
+username: your-username
+```
+
+**Usage:**
+
```

Comment:
I think this should be done in another PR.

Reply:
Is that the only change that you request?

Reply:
I think so, yes